data/articles/progress-bar.md

```diff
@@ -0,0 +1,395 @@
+---
+id: 11
+author: "kkpiotrowski"
+topics: ["hanami", "htmx", "sidekiq", "rom-rb", "redis", "sequel"]
+title: "Hanami and HTMX progress bar"
+excerpt: "<<Summary>>"
+videoId: null
+published: false
+publishedAt: 
+modifiedAt: 
+aliases: ['HMEP011']
```

Comment:
FYI: HME is an alias for `episode`, `HMA` is an alias for article. Could you actually move this to the `episodes` folder? I'd love to make a video out of it!

Reply:
Sure, will do